### PR TITLE
APIGOV-26852 - add tls config to kong client

### PR DESCRIPTION
**Pull request by jcollins-axway**

### CODE REVIEW

**Review by vivekschauhan — APPROVED** [review]

**Comment by dgghinea** [comment]
shouldn't there be helm changes as well ?

File: README.md [hunk 1]
```diff
@@ -67,29 +67,34 @@ On startup the Kong traceability agent first validates that it is able to connec
...
+| **KONG_ADMIN_AUTH_APIKEY_VALUE**       | The API Key value the agent will use when authenticating                                                                                                                                                                                           |
+| **KONG_ADMIN_AUTH_BASICAUTH_USERNAME** | The HTTP Basic username that the agent will use when authenticating                                                                                                                                                                                |
+| **KONG_ADMIN_AUTH_BASICAUTH_PASSWORD** | The HTTP Basic password that the agent will use when authenticating                                                                                                                                                                                |
+| **KONG_ADMIN_SSL_NEXTPROTOS**          | An array of strings. It is a list of supported application level protocols, in order of preference, based on the ALPN protocol list. Allowed values are: h2, http/1.0, http/1.1, h2c                                                               |
```
Comment (dgghinea) [inline README.md:79]: wouldn't these changes under "ssl" require some validate config updates as well ?

Comment (dgghinea) [reply README.md:79]: nevermind, I saw this comes from the SDK so I assume this is not required

**Comment by jcollins-axway** [comment]
> shouldn't there be helm changes as well ?
good call!

**Review by vivekschauhan — APPROVED** [review]

**Review by dgghinea — APPROVED** [review]

**Review by vivekschauhan — APPROVED** [review]

**Pull request merged**